saml/src/main/java/com/linecorp/armeria/server/saml/InvalidSamlRequestException.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.server.saml;
+
+import javax.annotation.Nullable;
+
+/**
+ * Indicates that a SAML request is not valid so that its problem should be propagated to the sender.
```

Comment:
nit: probably we can remove the clause after `not valid`?

Reply:
Yeah, it'd be better to remove that. Thanks!